taxi/views.py

```diff
@@ -27,13 +29,39 @@
     return render(request, "taxi/index.html", context=context)
 
 
+# -------Manufacturer----------------------------------------------------------
+
+
 class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     model = Manufacturer
     context_object_name = "manufacturer_list"
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
```

Comment:
Ensure that the template `taxi/manufacturer_form.html` exists and is correctly implemented for creating and updating manufacturers.